plugin/src/main/groovy/com/novoda/staticanalysis/EvaluateToolViolationsTask.groovy

```diff
@@ -0,0 +1,20 @@
+package com.novoda.staticanalysis
+
+import org.gradle.api.DefaultTask
+import org.gradle.api.tasks.TaskAction
+
+class EvaluateToolViolationsTask extends DefaultTask {
+
+    Closure<ViolationsEvaluator> evaluator
+    Violations toolViolations
+
+    EvaluateToolViolationsTask() {
+        group = 'verification'
+        description = "Evaluate violations against penaltyExtension thresholds."
+    }
+
+    @TaskAction
+    void run() {
+        evaluator().evaluate([toolViolations] as Set)
```

Comment:
`evaluate` expects a `Set` and, to keep changes small, I'm creating a Set with a single object to pass it to `evaluate`